public/js/modules.js

```diff
@@ -983,6 +1052,13 @@ NotGate.prototype.customDraw = function () {
     ctx.stroke();
 
 }
+//translate to not for single bit, assign for multi bit
+NotGate.prototype.generateVerilog = function () {
+    if (this.bitWidth == 1) 
+        return "not " + this.output1.verilogLabel + "(" + this.output1.verilogLabel + ", " + this.inp1.verilogLabel + ");"
```

Comment:
`this.output1.verilogLabel` cannot be used as `not` element right? as it might also be a `wire`?